src/__experimental__/components/date/date.spec.js

```diff
@@ -214,6 +214,24 @@ describe('Date', () => {
     });
   });
 
+  describe('when there is more than one date input, and we switch between them ', () => {
+    it('we should see only one date picker visible ', () => {
```

Comment:
Perhaps we could expand on this test so that it also checks if the other input has been focused. The 'it' block description does not go in pair with what is being tested.

Reply:
After changes I've made the tests are no longer necessary